presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -411,7 +411,13 @@ Property Name                         Description
 
 ``hive.s3.multipart.min-file-size``   Minimum file size before multi-part upload to S3 is used.   ``16 MB``
 
-``hive.s3.multipart.min-part-size``   Minimum multi-part upload part size.                        ``5 MB``
+``hive.s3.multipart.min-part-size``   Minimum multi-part upload part size. Can also be set via    ``10MB``
```

Comment:
```suggestion
``hive.s3.multipart.min-part-size``   Minimum multi-part upload part size. Can also be set using  ``10MB``
```
Do not use Latin abbreviations. For context and suggestions, see the GitLab [recommended word list entry for via](https://docs.gitlab.com/ee/development/documentation/styleguide/word_list.html#via). 

(I tested this, and discovered I needed to delete two blank spaces between "using" and "``10MB``" or the extra characters in "using" threw off the format for the Default value column.)

Reply:
fixed, thanks!